Orange/data/io.py

```diff
@@ -807,7 +807,11 @@ class PickleReader(FileFormat):
 
     def read(self):
         with open(self.filename, 'rb') as f:
-            return pickle.load(f)
+            table = pickle.load(f)
+            if not isinstance(table, Table):
+                raise Exception("Not a table.")
```

Comment:
Change the message to `"file does not contain a data table"`. It's more informative and, like most Python exceptions, starts with a lower case and has no period.